placement/storage/helper.go

```diff
@@ -155,6 +156,16 @@ func (h *stagedPlacementHelper) GenerateProto(p placement.Placement) (proto.Mess
 		return nil, err
 	}
 
+	var lastCutoverNanos int64
+	if len(ps) > 0 {
+		lastCutoverNanos = ps[len(ps)-1].CutoverNanos()
+	}
+	// When the placement has a valid cutover time, the new placement must be scheduled after existing placements.
+	if lastCutoverNanos != 0 && lastCutoverNanos >= p.CutoverNanos() {
+		return nil, fmt.Errorf("invalid placement: the cutover nanos is %d, must be later than %d",
```

Comment:
nit: `invalid placement: cutover nanos %d must be later than last placement cutover nanos %d`